src/ReactiveUI/Platforms/uap10.0/DependencyObjectObservableForProperty.cs

```diff
@@ -2,23 +2,13 @@
 // The .NET Foundation licenses this file to you under the MS-PL license.
```

Comment:
This file is only in UAP, so NETFX_CORE is always defined, which makes it a bit cleaner.